app/allocation/controllers/view.js

```diff
@@ -8,8 +8,10 @@ function viewAllocation(req, res) {
   const bannerStatuses = ['cancelled']
   const movesWithoutProfile = moves.filter(move => !move.profile)
   const movesWithProfile = moves.filter(move => move.profile)
+  const { location_type: locationType } = req.session.currentLocation
```

Comment:
Could we swap this for [`req.location`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/main/common/middleware/set-location.js). This will prevent the bug that Robin fixed where if you view a location that isn't actually your "current location" it will still function as expected.

It will allow the location to be taken from the URL first and then current location if one doesn't exist.